tests/test_interactive.py

```diff
@@ -194,12 +195,12 @@ def test_undo_latest_msg(mock_endpoint):
     httpretty.register_uri(httpretty.PUT, replace_url)
 
     httpretty.enable()
-    online._undo_latest(sender_id, mock_endpoint)
+    interactive._undo_latest(sender_id, mock_endpoint)
     httpretty.disable()
 
     b = httpretty.latest_requests[-1].body.decode("utf-8")
 
-    # this should be the events the online call send to the endpoint
+    # this should be the events the interactive call send to the endpoint
```

Comment:
This one should still be "online" unless I'm not mistaken

Reply:
I'm not sure I follow. My idea was to keep `online` for operations which really do online training, e.g. finetuning an existing model, and calling everything else `interactive`

Reply:
cool, thanks! I'm not sure why test coverage decreased since I didn't actually change any code. Also the codeclimate checks aren't about anything introduced here.